src/features/bookReader/components/ReaderDownloadButton.tsx

```
@@ -0,0 +1,20 @@
+import { Download } from 'lucide-react'
+import { ReaderMenuButton } from '@/features/bookReader/components/ReaderMenuButton'
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip'
+
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+export type ReaderDownloadButtonProps = {}
+
+export function ReaderDownloadButton({}: ReaderDownloadButtonProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)